include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -6639,6 +6780,97 @@ FEEvaluationAccess<dim, 1, Number, is_face, VectorizedArrayType>::
 
 
 
+template <int dim, typename Number, bool is_face, typename VectorizedArrayType>
+inline DEAL_II_ALWAYS_INLINE void
+FEEvaluationAccess<dim, 1, Number, is_face, VectorizedArrayType>::
+  submit_hessian(const Tensor<2, dim, VectorizedArrayType> hessian_in,
+                 const unsigned int                        q_point)
+{
+  Assert(this->cell != numbers::invalid_unsigned_int, ExcNotInitialized());
+  AssertIndexRange(q_point, this->n_quadrature_points);
```

Comment:
I think this function is expensive enough on its own that we could simply forward the implementation to the generic function `FEEvaluationBase::submit_hessian`, just as `submit_normal_derivative` does, in order to avoid repeating a lot of code.